src/extension.ts

```diff
@@ -135,6 +122,26 @@ export async function activateInternal(context: vscode.ExtensionContext) {
     });
 }
 
+export async function associateXmlSchema(context: vscode.ExtensionContext): Promise<void> {
+    const shouldAssociateSchema = process.env.APIM_AUTO_ASSOCIATE_SCHEMA?.toLowerCase() !== 'false';
```

Comment:
Is this feature for end users to control xml schema association?
If yes, would it be better to user vs code settings instead of env?

Reply:
I just add this environment variable in case any user complain we shouldn't bind the schema automatically. In that case, we could use this env as a workaround for users. I think this is a corner case instead of a feature for users. So didn't add it to vs code settings to reduce the effort.